lib/webdriver.js

```diff
@@ -94,9 +94,19 @@ Webdriver.prototype._init = function() {
 
   var url = httpUtils.buildInitUrl(this.configUrl);
 
+  var data = {};
+
   // building request
-  var data = {desiredCapabilities: _desired};
+  if (_desired.allowW3C || _desired.forceW3C) {
```

Comment:
I think it should be opt-in for now because WD still doesn't have _full_ support for all of the W3C endpoints.